tests/testutils.py

```diff
@@ -0,0 +1,49 @@
+"""Helpers for the various test_*.py files."""
+import os
+import sys
+import unittest
+
+TESTS = os.path.dirname(os.path.realpath(__file__))
+ROOT = os.path.normpath(os.path.join(TESTS, ".."))
+DATA = os.path.normpath(os.path.join(TESTS, "data"))
+
+# Prepend the tree's root folder to the module searchpath so we find zeekscript
+# via it. This allows tests to run without package installation. (We do need a
+# package build though, so the .so bindings library gets created.)
+sys.path.insert(0, ROOT)
+
+import zeekscript  # pylint: disable=wrong-import-position
```

Comment:
Having this test module provide `zeekscript` which works even when not installed allows us to fix an issue around test files importing `zeekscript` directly, but e.g., tools like `pylint` not finding it. This currently breaks e.g., pre-commit checks on single test files.

We should get rid of the direct imports there and instead `from testutils import zeekscript` which should always work.